app/data/projects.json

```diff
@@ -4,2053 +4,2657 @@
     "projectDescription": "Automatic bug detection and reporting tool",
     "projectRepository": "https://github.com/abrt",
     "projectWebsite": "https://abrt.readthedocs.org/en/latest/",
-    "category": "Development"
+    "category": "Development",
```

Comment:
Can you tidy up the spacing in this file for me when you get a chance?

Reply:
Hi Cassondra, would you please be able to elaborate what spacing changes you would need?

Reply:
It might have just been the diff view I was seeing.  Just make sure that the files have consistent spacing.  You can run it through a prettier tool if that is helpful.  I might add a todo for myself to incorporate prettier in the future so it's not even a concern.